jcl/src/openj9.traceformat/share/classes/com/ibm/jvm/traceformat/TraceFormat.java

```diff
@@ -355,21 +356,23 @@ public static void main(String[] args) throws Exception {
 				thread = current;
 				output.println(formatted.toString());
 			}
-			
+
 			/* print percentage */
 			if (context.getTotalRecords() != recordsProcessed) {
 				recordsProcessed = context.getTotalRecords();
 				float processedMbytes = ((float) (context.getTotalRecords() * context.getRecordSize())) / (1024 * 1024);
 				if (processedMbytes % 10 == 0) {
-					int percent = (int) ((processedMbytes * 100) / (totalBytes/(1024*1024)));
+					int percent = (int) ((processedMbytes * 100) / (totalBytes / (1024 * 1024)));
```

Comment:
This subtly depends on the fact that `totalBytes` must be more than 10MB, but it doesn't need to:
```
    int percent = (int) ((100 * bytesProcessed) / totalBytes);
```

Reply:
updated.